src/jsSSE.php

```diff
@@ -2,9 +2,7 @@
 
 namespace atk4\ui;
 
-/*
- * Implements a class that can be mapped into arbitrary JavaScript expression.
- */
+// Implements a class that can be mapped into arbitrary JavaScript expression.
```

Comment:
Use `/**` (two stars) for proper phpdoc

Reply:
Done! Let's merge this in so it get include in next release.